buddyboss/buddyboss-platform
# #1945 - Fix Profile field names not escaped, backslashes added

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2236 .

### How to test the changes in this Pull Request:

1. BuddyBoss > Profiles > Click **Edit** button on any field
2. Update the Title like **I'm field**
3. Click on update and see the backslash is no more added

### Proof Screenshots or Video
https://vimeo.com/593235504/411f8aa96b


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
